store/issue.go

```diff
@@ -538,13 +538,32 @@ func (*Store) findIssueImpl(ctx context.Context, tx *sql.Tx, find *api.IssueFind
 		args = append(args, *v)
 		args = append(args, *v)
 	}
+	if v := find.ExcludedIssueInArchiveProject; v != nil && *v {
```

Comment:
similar to https://sourcegraph.com/github.com/bytebase/bytebase/-/commit/62a03474e8bea42aa50d0829560bb36be695ce60?visible=1